src/olympia/accounts/views.py

```diff
@@ -118,11 +119,11 @@ def find_user(identity):
         raise
 
 
-def register_user(request, identity):
+def register_user(sender, request, identity):
     user = UserProfile.objects.create_user(
         email=identity['email'], username=None, fxa_id=identity['uid'])
     log.info('Created user {} from FxA'.format(user))
-    login(request, user)
+    login_user(sender, request, user, identity)
```

Comment:
Mmm, we're calling `update_user` now immediately after creating it.  Though I suppose that's okay because nothing will need updating.

Reply:
Yeah, that's what I figured. In fact, I do think having it call `update_user()` is a good thing because if we add more stuff to it in the future, we'll probably want that for new users as well... hopefully...